grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcJsonMarshaller.java

```diff
@@ -39,19 +39,62 @@
 
     /**
      * Returns a newly-created {@link GrpcJsonMarshaller} which serializes and deserializes a {@link Message}
-     * served by the {@linkplain ServiceDescriptor service}.
+     * served by the {@linkplain ServiceDescriptor service}. This implementation internally uses
+     * <a href="https://github.com/curioswitch/protobuf-jackson">protobuf-jackson</a>
+     * to serialize and deserialize messages.
      */
     static GrpcJsonMarshaller of(ServiceDescriptor serviceDescriptor) {
-        return builder().build(serviceDescriptor);
+        return ofJackson(serviceDescriptor);
     }
 
     /**
-     * Returns a new {@link GrpcJsonMarshallerBuilder}.
+     * Returns a new {@link GrpcJsonMarshallerBuilder}. This implementation internally uses
+     * <a href="https://github.com/curioswitch/protobuf-jackson">protobuf-jackson</a>
+     * to serialize and deserialize messages.
      */
     static GrpcJsonMarshallerBuilder builder() {
+        return builderForJackson();
+    }
+
+    /**
+     * Returns a newly-created {@link GrpcJsonMarshaller} which serializes and deserializes a {@link Message}
+     * served by the {@linkplain ServiceDescriptor service}. This implementation internally uses
+     * <a href="https://github.com/curioswitch/protobuf-jackson">protobuf-jackson</a>
+     * to serialize and deserialize messages.
+     */
+    static GrpcJsonMarshaller ofJackson(ServiceDescriptor serviceDescriptor) {
```

Comment:
Doesn't look like there's really a need for `ofJackson` since it's the same as the default